easybuild/tools/github.py

```diff
@@ -348,11 +349,14 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
 
     if (stable or pr_data['merged']) and not closed:
         # whether merged or not, download develop
-        path = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+        dl_path = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
```

Comment:
I would drop the `path=path`, which will make `download_repo` download `develop` into a separate tmpdir

otherwise we'll have both the `easybuild-easyconfigs-develop/easybuild/easyconfigs` and `a`..`z` dirs intermixed in a single parent directory...